src/vivarium_public_health/disease/state.py

```diff
@@ -102,110 +120,124 @@ def _transition_side_effect(self, index, event_time):
         -------
-        vivarium.framework.state_machine.Transition
+        RateTransition
             The created transition object.
-
         """
-        transition_map = {"rate": RateTransition, "proportion": ProportionTransition}
-
-        if not source_data_type:
-            return super().add_transition(output, **kwargs)
-        elif source_data_type in transition_map:
-            t = transition_map[source_data_type](self, output, get_data_functions, **kwargs)
-            self.transition_set.append(t)
-            return t
-        else:
-            raise ValueError(f"Unrecognized data type {source_data_type}")
+        transition = RateTransition(self, output, get_data_functions, **kwargs)
+        self.add_transition(transition)
+        return transition
 
-
-class SusceptibleState(BaseDiseaseState):
-    def __init__(self, cause, *args, **kwargs):
-        super().__init__(cause, *args, name_prefix="susceptible_to_", **kwargs)
-
-    def add_transition(
+    def add_proportion_transition(
         self,
-        output: State,
-        source_data_type: str = None,
+        output: "BaseDiseaseState",
         get_data_functions: Dict[str, Callable] = None,
         **kwargs,
-    ) -> Transition:
-        if source_data_type == "rate":
-            if get_data_functions is None:
-                get_data_functions = {
-                    "incidence_rate": lambda builder, cause: builder.data.load(
-                        f"{self.cause_type}.{cause}.incidence_rate"
-                    )
-                }
-            elif "incidence_rate" not in get_data_functions:
-                raise ValueError("You must supply an incidence rate function.")
-        elif source_data_type == "proportion":
-            if "proportion" not in get_data_functions:
-                raise ValueError("You must supply a proportion function.")
-
-        return super().add_transition(output, source_data_type, get_data_functions, **kwargs)
-
-
-class RecoveredState(BaseDiseaseState):
-    def __init__(self, cause, *args, **kwargs):
-        super().__init__(cause, *args, name_prefix="recovered_from_", **kwargs)
-
-    def add_transition(
+    ) -> ProportionTransition:
+        """Builds a ProportionTransition from this state to the given state.
+
+        Parameters
+        ----------
+        output
+            The end state after the transition.
+
+        get_data_functions
+            map from transition type to the function to pull that transition's data
+
+        Returns
+        -------
+        RateTransition
+            The created transition object.
+        """
+        if "proportion" not in get_data_functions:
+            raise ValueError("You must supply a proportion function.")
+
+        transition = ProportionTransition(self, output, get_data_functions, **kwargs)
+        self.add_transition(transition)
+        return transition
+
+
+# TODO is there a better name for the union of susceptible and recovered?
```

Comment:
ChatIHME suggests "non-diseased" is an epidemiological term of art for this, and I think it is correct based on a quick follow-up